src/pdsfile/holdings_maintenance/pds3/update_holdings_for_new_metadata.sh

```
@@ -28,15 +28,16 @@ rm -rf "$HOLDINGS/archives-metadata/$VOLSET"
 rm -f "$HOLDINGS"/checksums-archives-metadata/${VOLSET}_*
 rm -rf "$HOLDINGS/checksums-metadata/$VOLSET"
 rm -rf "$HOLDINGS/_indexshelf-metadata/$VOLSET"
-rm -rf "$HOLDINGS/_infoshelf-archives-metadata/$VOLSET"
+rm -f "$HOLDINGS"/_infoshelf-archives-metadata/${VOLSET}_info.*
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)